index.js

```diff
@@ -19,9 +19,31 @@ const manageRecipes = async () => {
     await Recipe.deleteMany();
 
     // Run your code here, after you have insured that the connection was made
+    await Recipe.create({
+      title: "Hot Dog Fingers",
+      level: "UltraPro Chef",
+      ingredients: ["5 hot dogs", "5 human fingers", "100g of cut off nails"],
+      cuisine: "Anthropofagic",
+      dishType: "snack",
+      duration: 120,
+      creator: "Chef Hannibal",
+    });
+    console.log("Hot Dog Fingers");
   } catch (error) {
     console.log(error);
   }
+
+  await Recipe.insertMany(data);
+  data.forEach((Recipe) => console.log(Recipe.title));
+
+  await Recipe.findOneAndUpdate(
+    { title: "Rigatoni alla Genovese" },
+    { duration: 100 }
+  );
+  console.log("Great Success!");
```

Comment:
Same thing here, but now you're updating, not creating. So there is one extra step that is `{new: true}` if you do not include this inside the arguments, the code will work but it will return you the old version of the recipe instead of the updated one:

```
    const updateRecipe = await Recipe.findOneAndUpdate(
      { title: "Rigatoni alla Genovese" },
      { duration: 100 },
      { new: true }
    );
    console.log(`Recipe Updated: ${updateRecipe}`);
```